Sources/MapboxNavigation/CPRouteChoice.swift

```diff
@@ -0,0 +1,34 @@
+import CarPlay
+import MapboxDirections
+
+@available(iOS 12.0, *)
+extension CPRouteChoice {
+    
+    struct RouteResponseUserInfo {
+        
+        static let key = "com.mapbox.navigation.cpRouteChoice.routeResponse"
```

Comment:
Let’s use `Bundle.mapboxNavigation.bundleIdentifier` to compose this key for consistency with other keys in the codebase:

https://github.com/mapbox/mapbox-navigation-ios/blob/5018a7f4b32e9aeebf20a32f2cb6169c43b89723/Sources/MapboxNavigation/NavigationMapViewIdentifiers.swift#L5